### PR TITLE
Make no tradable

### PR DESCRIPTION
**Pull request by bthaile**
<https://github.com/AugurProject/augur/issues/3698>

expose NO for trading. cleaned up some logic. verify NO is tradable on YesNo and user gets long/short positions

noticed that user created market infos were not getting loaded.

to test:
create Yes No market with orders
when the market is created open market liquidity and create orders
when the market liquidity is created nav to market and create positions.
verify user can create positions on No and Yes outcomes.

image

image

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🌻

 This link will always point to this PR's latest changes: https://peek.run/93ph3m1

File: packages/augur-ui/src/modules/markets/actions/load-create-market-history.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { AppState } from 'store';
 import { ThunkDispatch } from 'redux-thunk';
 import { Action } from 'redux';
 import { updateMarketsData } from './update-markets-data';
-import { keyBy } from 'utils/key-by';
+import { keyMarketInfoCollectionByMarketId } from 'utils/convert-marketInfo-marketData';
```
Comment (bconfortin) [inline packages/augur-ui/src/modules/markets/actions/load-create-market-history.ts]: Much better

File: packages/augur-ui/src/utils/convert-marketInfo-marketData.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ import {
 import { convertUnixToFormattedDate } from './format-date';
 import { formatPercent, formatDai, formatNone, formatNumber, formatAttoRep } from './format-number';
 import { createBigNumber } from './create-big-number';
+import { keyBy } from './key-by';
```
Comment (bconfortin) [inline packages/augur-ui/src/utils/convert-marketInfo-marketData.ts]: @bthaile 
Wasn't that renamed to `keyMarketInfoCollectionByMarketId`?

Comment (bthaile) [reply packages/augur-ui/src/utils/convert-marketInfo-marketData.ts]: gave a better name on later commit.

**Review by bconfortin — APPROVED** [review]

**Comment by bthaile** [comment]
closing putting on integration branch to merge up

**Pull request closed without merging**